@axiomhq/logging@0.2.0 peer-dep on @axiomhq/js is exact-pinned ("1.4.0") — breaks Dependabot/npm install

## Summary

`@axiomhq/logging@0.2.0` declares a strict, exact-version peer dependency on `@axiomhq/js@"1.4.0"`. Any install that resolves `@axiomhq/js` to a newer version (e.g. `1.5.x`, `1.6.0`) fails with `ERESOLVE`. This affects all consumers using Dependabot, Renovate, or a clean `npm install` against a lockfile-less checkout (such as Vercel's build step in some configurations).

## Current behavior

From `node_modules/@axiomhq/logging/package.json` (version `0.2.0`):

```json
{
  "peerDependencies": {
    "@axiomhq/js": "1.4.0"
  }
}
```

The `"1.4.0"` selector is an **exact** version match (no caret, no range). Installing `@axiomhq/logging@0.2.0` alongside any other version of `@axiomhq/js` throws:

```
npm error ERESOLVE could not resolve
npm error While resolving: @axiomhq/logging@0.2.0
npm error Found: @axiomhq/js@1.6.0
npm error Could not resolve dependency:
npm error peerOptional @axiomhq/js@"1.4.0" from @axiomhq/logging@0.2.0
```

Note: `@axiomhq/js` is also listed in peerDependenciesMeta as `{ "optional": true }`, but the exact-version constraint still triggers ERESOLVE whenever a consumer installs a different version of `@axiomhq/js`.

## Expected behavior

The peer range should follow standard semver-compatible ranges, e.g.:

```json
{
  "peerDependencies": {
    "@axiomhq/js": "^1.4.0"
  }
}
```

This would allow patch/minor bumps of `@axiomhq/js` (1.4.x, 1.5.x, 1.6.x) without forcing consumers to also upgrade `@axiomhq/logging` in lockstep.

## Reproduction

```bash
mkdir axiom-repro && cd axiom-repro
npm init -y
npm install @axiomhq/logging@0.2.0 @axiomhq/js@1.6.0
# → ERESOLVE
```

## Impact

- **Dependabot/Renovate users:** When Dependabot opens a PR bumping `@axiomhq/js` (e.g. `1.4.0 → 1.6.0`), `npm ci` in CI passes because it respects the lockfile — but downstream `npm install` (used by some build systems, including certain Vercel configurations) re-resolves peer deps strictly and fails.
- **Fresh installs:** Any new developer or CI runner that does `npm install` without an existing lockfile hits the error immediately.
- **Invisible to CI, visible in production deploys:** The asymmetry between `npm ci` (lockfile-respecting) and `npm install` (strict peer resolution) means the issue can ship to production before being noticed.

## Workaround (current)

Consumers must pin `@axiomhq/js` to exactly `1.4.0` in their own `package.json`:

```json
{
  "dependencies": {
    "@axiomhq/js": "1.4.0",
    "@axiomhq/logging": "^0.2.0"
  }
}
```

This is brittle — it requires all consumers to know about the peer-lock and manually sync versions whenever Axiom ships a new `@axiomhq/logging`.

## Suggested fix

Either of the following would resolve this:

1. Widen the peer range in `@axiomhq/logging` to `"^1.4.0"` (recommended).
2. Release a patch version of `@axiomhq/logging` with a compatible peer range whenever a new `@axiomhq/js` ships.

Option 1 is the standard npm convention and requires no ongoing maintenance coordination between the two packages.

## Environment

- Node: v25.9.0
- npm: 11.12.1
- `@axiomhq/logging`: `0.2.0`
- `@axiomhq/js`: attempted `1.5.x`, `1.6.0`

--- a/packages/logging/package.json
+++ b/packages/logging/package.json
@@ -62,7 +62,7 @@
     "vite": "^7.3.2"
   },
   "peerDependencies": {
-    "@axiomhq/js": "^1.4.0"
+    "@axiomhq/js": "^1.6.0"
   },
   "peerDependenciesMeta": {
     "@axiomhq/js": {

--- a/packages/logging/package.json
+++ b/packages/logging/package.json
@@ -62,7 +62,7 @@
     "vite": "^7.3.2"
   },
   "peerDependencies": {
-    "@axiomhq/js": "^1.6.0"
+    "@axiomhq/js": "^1.0.0"
   },
   "peerDependenciesMeta": {
     "@axiomhq/js": {

--- a/packages/logging/package.json
+++ b/packages/logging/package.json
@@ -62,7 +62,7 @@
     "vite": "^7.3.2"
   },
   "peerDependencies": {
-    "@axiomhq/js": "workspace:*"
+    "@axiomhq/js": "^1.4.0"
   },
   "peerDependenciesMeta": {
     "@axiomhq/js": {

